.env.example

```diff
@@ -1,24 +1,32 @@
 # Get your Plaid API keys from the dashboard: https://dashboard.plaid.com/account/keys
```

Comment:
i know that you didn't break this, but this url is actually wrong -- it should be https://dashboard.plaid.com/team/keys -- while you're at it would you mind updating it?

Reply:
done!

Reply:
Thanks for the review Alex! 
I don't have permissions to merge, so if this looks good to you, can you please go ahead and merge it?